core/controllers/questions_list_test.py

```diff
@@ -80,18 +80,19 @@ def test_get_questions_succeeds(self):
         self.login(self.ADMIN_EMAIL)
         with self.swap(constants, 'NUM_QUESTIONS_PER_PAGE', 2):
             json_response = self.get_json(
-                '%s/%s,%s?cursor=' % (
+                '%s/%s,%s?offset=0' % (
                     feconf.QUESTIONS_LIST_URL_PREFIX,
                     self.skill_id, self.skill_id_2
                 ))
             question_summary_dicts = json_response['question_summary_dicts']
             self.assertEqual(len(question_summary_dicts), 2)
-            next_start_cursor = json_response['next_start_cursor']
+            more = json_response['more']
+            self.assertTrue(more)
             json_response = self.get_json(
-                '%s/%s,%s?cursor=%s' % (
+                '%s/%s,%s?offset=%s' % (
                     feconf.QUESTIONS_LIST_URL_PREFIX,
                     self.skill_id, self.skill_id_2,
-                    next_start_cursor
+                    4
```

Comment:
Put 4 into the raw string, you don't need to interpolate it since it's a constant.

Reply:
Done